src/global_const.js

```diff
@@ -2249,13 +2249,13 @@ var supportAbilities = {
         "name": "1回攻撃と2回攻撃時に火属性追加ダメージ発生(1回:80%、 2回:30%)。(スツルム)",
         "type": "additionalDamageXA",
         "range": "own",
-        "additionalDamageXA": [0.8, 0.3, 0.0]
+        "value": [0.8, 0.3, 0.0]
```

Comment:
NOTE: my previous PR changed `totals[key]["additionalDamageXA"] = support.value;`
so this is FIX for the change of support field. noted in PR comment, why renamed to "support.value"